### PR TITLE
Update v1.2 readme [CI SKIP]

### PR DESCRIPTION
**Pull request by guangbochen**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
Update readme doc for v1.2 changes

**Solution:**
Update v1.2 ReadME

**Related Issue:**
https://github.com/harvester/harvester/issues/4294

**Note:**
Preview on: https://github.com/guangbochen/harvester/blob/v1.2-readme/README.md
and merge it after the v1.2.0 official tag is ready.

### CODE REVIEW

**Review by bk201 — APPROVED** [review]
Thanks for updating the screenshots.

**Review by yasker — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -52,31 +50,51 @@ You can use the ISO to install Harvester directly on the bare-metal server to fo
 
 To get the Harvester ISO, download it from the [Github releases.](https://github.com/harvester/harvester/releases)
 
-During the installation, you can either choose to form a new cluster, or join the node to an existing cluster.
+During the installation you can either choose to **create a new Harvester cluster** or **join the node to an existing Harvester cluster**.
```
Comment (LucasSaintarbor) [inline README.md]:
```suggestion
During the installation, you can either choose to **create a new Harvester cluster** or **join the node to an existing Harvester cluster**.
```

File: README.md [hunk 2]
```diff
@@ -52,31 +50,51 @@ You can use the ISO to install Harvester directly on the bare-metal server to fo
...
-1. (Optional) If you need to customize the host with a cloud-init config, enter the HTTP URL here.
-1. After confirming the installation options, Harvester will be installed to your host. The installation may take a few minutes to be complete.
-1. Once the installation is complete, it will restart the host. After the restart, the Harvester console containing the management URL and status will be displayed. You can Use `F12` to switch between the Harvester console and the Shell.
-1. The default URL of the web interface is `https://your-virtual-ip`.
+1. (Optional) You can choose to import SSH keys by providing `HTTP URL`. For example, your GitHub public keys `https://github.com/<username>.keys` can be used.
+1. (Optional) If you need to customize the host with a [Harvester configuration](./harvester-configuration.md) file, enter the `HTTP URL` here.
+1. Review and confirm your installation options. After confirming the installation options, Harvester will be installed to your host. The installation may take a few minutes to be complete.
```
Comment (LucasSaintarbor) [inline README.md]:
```suggestion
1. Review and confirm your installation options. After confirming the installation options, Harvester will be installed on your host. The installation may take a few minutes to complete.
```

**Review by LucasSaintarbor — APPROVED** [review]
Small NITs. LGTM 👍

**Pull request merged**